File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C+DS/codes/Recursion/p29.c
# include <stdio.h>
# include <stdbool.h>
int count=0 ;
bool demo (int arr[],int size) {
	if (size==1) 
		return false ;
	if (count>=2) 
		return true ;
	if (arr[size-1]%2==0) {
		count++ ;
		return demo(arr,size-1) ;
	}
	if (arr[size-1]%2!=0) 
		return demo(arr,size-1) ;
}
void main () {
	int size ;
	printf("Enter size of an array\n") ;
	scanf("%d",&size) ;

	int arr[size] ;
	printf("Enter elements in an array\n") ;
	for (int i=0 ; i<size ; i++) {
		scanf("%d",&arr[i]) ;
	}

	bool ret=demo(arr,size) ;
	if (ret==true) 
		printf("Found\n") ;
	else 
		printf("Not found\n") ;
}

File: java-main/java-main/DSA/Array/p29.java
class Demo{
	public static void main(String[]args){
		int arr[][]=new int [][]{{1,2,3,4},{5,6,7,8},{9,10,11,12},{13,14,15,16}};
		for(int i=0;i<arr.length;i++){
			for(j=i+1;j<arr.length;j++){
				int temp=arr[i][j];
				arr[i][j]=arr[j][i];
				arr[j][i]=temp;
			}
		}
		for(int i=0;i<arr.length;i++){
			for(j=0;j<arr.length;j++){
				System.out.println(arr[j][i]);
			}
		}
	}
}


